test/index.js

```diff
@@ -63,8 +63,15 @@ fixtures.valid.ec.forEach(function (f) {
   }
 
   test(f.message, function (t) {
-    var nSign = nCrypto.createSign(f.scheme)
-    var bSign = bCrypto.createSign(f.scheme)
+    var nSign, bSign
+    try {
+      nSign = nCrypto.createSign(f.scheme)
```

Comment:
only this one needs to be in the try catch

Reply:
moved outside